community/server/pom.xml

```diff
@@ -388,8 +388,8 @@
           <classpathScope>test</classpathScope>
           <systemProperties>
             <systemProperty>
-              <key>org.neo4j.server.properties</key>
-              <value>${basedir}/neo4j-home/conf/neo4j-server.properties</value>
+              <key>org.neo4j.config.file</key>
+              <value>${basedir}/neo4j-home/conf/neo4j.properties</value>
```

Comment:
@jakewins We'll be doing work later to change the way paths for stuff are specified. That'll be a good time to look at this.